### ISSUE TITLE
Crouton sticker for chromebooks

### ISSUE THREAD

**Issue by darcyparker** [dnschneid/crouton#1959]
Are there crouton stickers available?  Has anyone investigated printing them?  Is there SVG artwork available that could be used as a starting point?

**Comment by mark-hahn** [comment]
I'd buy one.  I noticed at the latest conference I attended, codeconf, that
everyone's laptops were covered with stickers.  It looked like a nascar
race.  I have one sticker for the Atom editor I use and contribute to.

**Comment by tedm** [comment]
I'll sell one.

**Comment by DennisLfromGA** [comment]
See [Dr.Inkcat](https://plus.google.com/+NicolasBoichat/posts/bRnKwr6XzjV)

**Comment by mark-hahn** [comment]
> See Dr.Inkcat

Do you have a link for that?

**Comment by DennisLfromGA** [comment]
[Yup](https://github.com/dnschneid/crouton/blob/master/host-ext/logo.svg)

**Comment by mark-hahn** [comment]
What should the background color be, White, Black, Gray?  I think white.

**Comment by mark-hahn** [comment]
we can also get transparent

If no one chimes in I'll get: Transparent.  Size: 2" by 2".

I'll give them away.  They are $10 for 20 stickers but it's the least I can
do for crouton.

**Comment by mark-hahn** [comment]
Ok, I've bought 20 2" by 2" stickers.  I'll get them in a week or so.

Unfortunately transparent vinyl was extra so I paid $22 for 20 of them.  I'll sell them for $1 each, mainly to make sure you are serious. See https://groups.google.com/forum/#!topic/crouton-central/93QnqJv19jQ for ordering details.

**Comment by tedm** [comment]
mark, I just bought 4 or 5, one for each chromebook, 1 for nexus 6,and 2 for each cats. But if you're running short, just send 4,and the cats will have to share.

**Comment by mark-hahn** [comment]
> if you're running short,

I'll just buy more.

> cats will have to share

You put stickers on your cats?  lol

**Comment by darcyparker** [comment]
@mark-hahn - thanks for taking the initiative to order these!  I just ordered a couple.

**Comment by DennisLfromGA** [comment]
> @mark-hahn - thanks for taking the initiative to order these! I just ordered a couple.

Me too - or should I say one. ;)

**Comment by mark-hahn** [comment]
They finally got here and look ok.  They'll go out tomorrow.

**Comment by tedm** [comment]
@mark-hahn Thanks! Got them today, they look great! Here's an image of one on my Intel Toshiba CB2: 
![crouton_logo](https://cloud.githubusercontent.com/assets/3884283/9291987/b9181e7e-4394-11e5-8c2a-cf21fa786b8f.jpg)

**Comment by mark-hahn** [comment]
That's looks better than mine does in real life.  So your chromebook is made out of wood?

**Comment by tedm** [comment]
Yes, ebony.

**Comment by darcyparker** [comment]
@mark-hahn - thanks. I just got back from Vacation and received mine too.

![darcy-croutonsticker](https://cloud.githubusercontent.com/assets/945058/9330378/0898b820-4587-11e5-9c63-fc55300665fc.jpg)

It's tricky putting on a transparent background sticker... I probably should have worn some disposable Nitrile or latex gloves.  Even though I cleaned the surface, let it dry and had clean hands, I managed to get some finger prints on the edge....   In the future if I get a little sticker I may use it to cover up the smudge I made.

If I didn't have other stickers on, it would have been nice to wrap the crouton logo around the Chrome logo on the laptop...
